packages/react-router/__tests__/useRoutes-test.tsx

```diff
@@ -114,7 +120,10 @@ describe("useRoutes", () => {
     TestRenderer.act(() => {
       renderer = TestRenderer.create(
         <MemoryRouter initialEntries={["/two"]}>
-          <NullRenderer routes={routes} location={{ pathname: "/three" }} />
+          <NullRenderer
+            routes={routes}
+            location={{ pathname: "/three", search: "", hash: "" }}
```

Comment:
The lack of search/hash in this test case was causing weird `console.warn` of `/threeundefinedundefined`